### PR TITLE
Add note about imageset not being upgradeable due to historical reasons

### LINKED ISSUE #64: Spec should explain why do we not upgrade when a request's initiator is `imageset`
This came up in #63: The spec makes an explicit carve-out for imageset and it's not immediately obvious why

CC @moztomer

### PR DESCRIPTION
**Pull request by carlosjoan91**
Fixes #64

### CODE REVIEW

**Comment by mozfreddyb** [comment]
looks good from our side, thanks.

File: index.bs [hunk 1]
```diff
@@ -282,6 +282,12 @@ type: dfn
           <li>
             <var>request</var>'s <a for="request">destination</a> is "<code>image</code>"
             and <var>request</var>'s <a for="request">initiator</a> is "<code>imageset</code>".
+
+            Note: For historical reasons, <code>imageset</code> is not upgraded. Prior to some
+            types of mixed content being upgraded, the spec divided mixed content into blockable
+            and optionally-blockable. <code>imageset</code> was part of the former, and when
+            upgrades were added, the decision was to not upgrade any content that was previously
+            defined as blockable.
```
Comment (annevk) [inline index.bs:290]: Nit:
```suggestion
            Note: For historical reasons, "<code>imageset</code>" is not upgraded. Prior to some
            types of mixed content being upgraded, the spec divided mixed content into blockable
            and optionally-blockable. "<code>imageset</code>" was part of the former, and when
            upgrades were added, the decision was to not upgrade any content that was previously
            defined as blockable.
```

File: index.bs [hunk 2]
```diff
@@ -282,6 +282,12 @@ type: dfn
           <li>
             <var>request</var>'s <a for="request">destination</a> is "<code>image</code>"
             and <var>request</var>'s <a for="request">initiator</a> is "<code>imageset</code>".
+
+            Note: For historical reasons, "<code>imageset</code>" is not upgraded. Prior to some
+            types of mixed content being upgraded, the spec divided mixed content into blockable
+            and optionally-blockable. "<code>imageset</code>" was part of the former, and when
```
Comment (estark37) [inline index.bs:288]: suggest adding "blockable and optionally-blockable, with blockable being preferred for any type of mixed content that wasn't prevalent."

**Comment by estark37** [comment]
LGTM with one suggestion

File: index.bs (hunk 2 above)
Comment (carlosjoan91) [reply index.bs]: Done, thanks for reviewing.

**Pull request merged**